### ISSUE TITLE
Problem with multiprocessing in predict.py

### ISSUE THREAD

**Issue by aweaver1fandm** [devanshkv/fetch#29]
**Describe the bug**
predict.py has issues when running with -n  greater than 1.  

**To Reproduce**
run predict with -n greater than 1 

I suspect the problem is with the following code starting on line 70 of predict.py

```python
 if args.nproc == 1:
        use_multiprocessing = True
        logging.info(f"Using multiprocessing with {args.nproc} workers")
    else:
        use_multiprocessing = False
```
Shouldn't the logic be:
if args.nproc > 1:

**Comment by devanshkv** [comment]
Yep, seems right, can you send a PR for the fix?

**Comment by devanshkv** [comment]
Thanks for your contribution. It was fixed with #31.
